src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -2141,10 +2141,16 @@ private unsafe ParsingError PrivateParseMinimal()
                         _flags |= Flags.AuthorityFound;
                         idx += 2;
                     }
+                    // There is no Authority component, save the Path index
+                    // Note: mailto is the only guy who is treated specially, should be not.
```

Comment:
What does "should be not" mean?

Reply:
It means I moved the comment without paying enough attention to what it said. Fixing this now 😄